README.md

```diff
@@ -372,6 +374,119 @@ const keyMap = {
 };
 ```
 
+#### Setting dynamic hotkeys
+
+`react-hotkeys` has basic support for setting dynamic hotkeys - i.e. letting the user set their own keyboard shortcuts. In your app, you can set up the necessary UI for [viewing the current keyboard shortcuts](#displaying-a-list-of-available-hot-keys), and opting to change them. You can then use the `recordKeyCombination` function to capture the keys the user wishes to use.
+
+`recordKeyCombination` accepts a callback function that will be called on the last `keyup` of the next key combination - immediately after the user has pressed the key combination they wish to assign. The callback then unbinds itself, so you do not have to worry about tidying up after it.
+
+`recordKeyCombination` returns a function you can call at any time after binding the listener, to cancel listening without waiting for the key combination to complete. 
+
+The callback function receives a single argument with the following schema:
+
+```javascript
+{
+  /**
```

Comment:
⬜️📚 Improve the comments/descriptions of the schema. Currently rather ambiguous/confusing.